Dockerfile

```diff
@@ -0,0 +1,17 @@
+# Dockerfile
+FROM php:8.3-cli
+WORKDIR /app
+
+# Install dependencies
+RUN docker-php-ext-install pdo pdo_mysql
+
+# Copy your project files to the container
+COPY . .
+
+# Install Composer dependencies
+RUN curl -sS https://getcomposer.org/installer | php && php composer.phar install
+
+# Run PHPUnit tests
+CMD ["vendor/bin/phpunit", "--bootstrap", "vendor/autoload.php", "tests/DockerContainerTest/ExampleTest.php"]
```

Comment:
Typically is better to leave entrypoint - now it will be locked by test, and you can't do anything production. 

Search how to run PHPunit insider docker by PHPStorm

Reply:
Deleted row CMD and made settings PHPunit insider docker by PHPStorm